### DISCUSSION TITLE
Floating terminal width

### DISCUSSION THREAD

**Discussion by inquisitev** [NvChad/NvChad#2935] [Q&A] [answered]
Hey All, what is the right way to set the width of the floating terminal? I want it to preferably by ~80% of my screen width and maybe slighly lower. 

ive tried rebinding the keys to a custom command including width
map({ "n", "t" }, "<A-i>", function()
    require("nvchad.term").toggle { pos = "float", id = "floatTerm", width = 0.8 }
end, { desc = "terminal toggle floating term" })

im just not sure what structure i can send in this toggle function and im kind of a nvim noob

**Comment by siduck** [comment, marked as answer]
you need to use float_opts for that or change height/width in chadrc https://github.com/NvChad/ui/blob/v2.5/lua/nvconfig.lua#L67

**Reply by inquisitev** [reply]
oh is see

Here is my change as an example for next person:
``` lua
map({ "n", "t" }, "<A-i>", function()
    require("nvchad.term").toggle { pos = "float", id = "floatTerm", float_opts={
        row = 0.35,
        col = 0.05,
        width = 0.9,
        height = 0.5
    }}
end, { desc = "terminal toggle floating term" })
```

**Reply by mrjxtr** [reply]
@inquisitev from you code snippet, I was able to make the floating term into the size that I want... but how do you adjust the with of the vertical term or the height of the horizontal term?

I am also a vim noob and as well so I really need some help for this if you can.

**Reply by siduck** [reply]
@mrjxtr  size, go read :h nvui.term

**Reply by mrjxtr** [reply]
> h nvui.term

Thanks... i've done it with this..

```lua
map({ "n", "t" }, "<A-v>", function()
    require("nvchad.term").toggle({
        pos = "vsp",
        id = "vertTerm",
        size = 0.4,
    })
end, { desc = "terminal toggle vertical term" })
```
